### PR TITLE
fix/CCN3-1708/remove-brackets

### PR DESCRIPTION
**Pull request by LRydeard**
## Description

Removed brackets from validation messages

## Todo's before PR

- [x] Branch has been rebased against develop
- [x] Unit tests passed (`make test`)
- [x] PR named appropriately - see [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Commits-guideline)
- [x] PR description describes the overall goals of PR commits
- [x] Templates have been checked against the relevant user-story

## Migrations

- [ ] Null fields in models specifies default value
- [ ] You generated and applied migrations (`make migrate`)
- [ ] You updated fixtures (`make export`)

## PR Todo's

Please refer to PR [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-submit-a-pull-request)

- [x] Specified reviewers (even if it's yourself)
- [x] Specified assignees (those who'll merge)
- [x] Specified labels

## PR review

Please refer to PR review [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-review-a-pull_request)

- [x] Code syntax formatting checked
- [x] Debug messages are absent

## PR merge

Select only one:

- [ ] Should be merged?
- [x] Should be rebased?
- [ ] Should be squashed?

Please refer to PR merge [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-merge-a-pull_request)

### CODE REVIEW

File: application/views/PITH_views/PITH_children_details.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def get(self, request):
         if application.application_status == 'FURTHER_INFORMATION':
             for index, form in enumerate(form_list):
                 form.error_summary_template_name = 'returned-error-summary.html'
-                form.error_summary_title = "There was a problem (Child " + str(index + 1) + ")"
+                form.error_summary_title = "There was a problem Child {0}".format(str(index + 1))
```
Comment (ricaduchateau) [inline application/views/PITH_views/PITH_children_details.py]: I'm not sure if the formatting for this looks right, i.e. `There was a problem Child 1`?

File: application/views/PITH_views/PITH_own_children_details.py [hunk 2]
```diff
@@ -42,7 +42,7 @@ def get(self, request):
         if application.application_status == 'FURTHER_INFORMATION':
             for index, form in enumerate(form_list):
                 form.error_summary_template_name = 'returned-error-summary.html'
-                form.error_summary_title = "There was a problem (Child " + str(index + 1) + ")"
+                form.error_summary_title = "There was a problem Child {0}".format(str(index + 1))
```
Comment (ricaduchateau) [inline application/views/PITH_views/PITH_own_children_details.py]: I'm not sure if the formatting for this looks right, i.e. `There was a problem Child 1`?

File: application/views/other_people.py [hunk 3]
```diff
@@ -87,7 +87,7 @@ def other_people_adult_details(request):
             form.check_flag()
             if application.application_status == 'FURTHER_INFORMATION':
                 form.error_summary_template_name = 'returned-error-summary.html'
-                form.error_summary_title = "There was a problem (Person " + str(i) + ")"
+                form.error_summary_title = "There was a problem Person {0}".format(str(i))
```
Comment (ricaduchateau) [inline application/views/other_people.py]: I'm not sure if the formatting for this looks right, i.e. `There was a problem Person 1`?

File: application/views/other_people.py [hunk 4]
```diff
@@ -142,11 +142,10 @@ def other_people_adult_details(request):
                 request.POST, id=application_id_local, adult=i, prefix=i, email_list=email_list)
             form.remove_flag()
             form_list.append(form)
-            form.error_summary_title = 'There was a problem with the details (Person ' + str(
-                i) + ')'
+            form.error_summary_title = 'There was a problem with the details Person {0}'.format(str(i))
```
Comment (ricaduchateau) [inline application/views/other_people.py]: I'm not sure if the formatting for this looks right, i.e. `There was a problem with the details Person 1`?

File: application/views/other_people.py [hunk 5]
```diff
@@ -142,11 +142,10 @@ def other_people_adult_details(request):
...
             if application.application_status == 'FURTHER_INFORMATION':
                 form.error_summary_template_name = 'returned-error-summary.html'
-                form.error_summary_title = "There was a problem (Person " + str(i) + ")"
+                form.error_summary_title = "There was a problem Person {0}".format(str(i))
```
Comment (ricaduchateau) [inline application/views/other_people.py]: I'm not sure if the formatting for this looks right, i.e. `There was a problem Person 1`?

File: application/views/other_people.py [hunk 6]
```diff
@@ -332,7 +331,7 @@ def other_people_summary(request):
                 'table_object': table,
                 'fields': other_adult_fields,
                 'title': name,
-                'error_summary_title': ('There was a problem (' + name + ')')
+                'error_summary_title': ('There was a problem {0}'.format(name))
```
Comment (ricaduchateau) [inline application/views/other_people.py]: I'm not sure if the formatting for this looks right, i.e. `There was a problem Joe Bloggs`?

File: application/views/other_people.py [hunk 7]
```diff
@@ -363,7 +362,7 @@ def other_people_summary(request):
                 'table_object': Table([child.pk]),
                 'fields': other_child_fields,
                 'title': name,
-                'error_summary_title': ('There was a problem (' + name + ')')
+                'error_summary_title': ('There was a problem {0}'.format(name))
```
Comment (ricaduchateau) [inline application/views/other_people.py]: I'm not sure if the formatting for this looks right, i.e. `There was a problem Joe Bloggs`?

File: application/views/your_children.py [hunk 8]
```diff
@@ -169,7 +169,7 @@ def __create_child_table(child):
         'table_object': table,
         'fields': child_fields,
         'title': child.get_full_name(),
-        'error_summary_title': "There was a problem with your children's details (" + child.get_full_name() + ")"
+        'error_summary_title': "There was a problem with your children's details {0}".format(child.get_full_name())
```
Comment (ricaduchateau) [inline application/views/your_children.py]: I'm not sure if the formatting for this looks right, i.e. `There was a problem with your children's details Joe Bloggs`?

File: application/views/other_people.py (hunk 4 above)
Comment (ricaduchateau) [reply application/views/other_people.py]: Hi @LRydeard, this still needs updating.

**Comment by LRydeard** [comment]
Hi @ricaduchateau,
I'm going to have make amendments to these changes again, Kashif has decided on a new format.
I'll leave a comment when the full changes are made.

**Comment by LRydeard** [comment]
Should be okay now @ricaduchateau ⛵

**Pull request merged**